selectel/resource_selectel_mks_nodegroup_v1_test.go

```diff
@@ -172,6 +173,7 @@ resource "selectel_mks_nodegroup_v1" "nodegroup_tf_acc_test_1" {
   enable_autoscale    = true
   autoscale_min_nodes = 2
   autoscale_max_nodes = 3
+  nodegroup_type = STANDARD
```

Comment:
If we declare this field as computed in the resource schema it should not be in manifest.
Also it breaks acceptance tests.
Please remove this line.